### PR TITLE
Add compile steps with babel

### PR DESCRIPTION
**Pull request by jbovenschen**
Also changed the `examples` folder to `examples/standalone` to support future react-dom / react-native binding examples

### CODE REVIEW

File: examples/standalone/package.json [hunk 1]
```diff
@@ -0,0 +1,30 @@
+{
+  "name": "standalone",
+  "version": "0.0.0",
+  "description": "Standalone pdf-react example",
+  "main": "index.js",
+  "private": true,
+  "scripts": {
+    "start": "./../../node_modules/.bin/babel-node ./example.jsx"
```
Comment (diegomura) [inline examples/standalone/package.json]: What do you think if we get rid of the `.jsx` for a more standard `.js`.
Not sure why I did it this way, but it's odd

Comment (jbovenschen) [inline examples/standalone/package.json]: Nice catch will change

**Comment by jbovenschen** [comment]
I merged it with the changes in the master branch, and renamed the file

**Comment by diegomura** [comment]
When conflicts are resolved it's ready to merge!

**Comment by jbovenschen** [comment]
@diegomura There are no conflicts anymore right?

**Comment by diegomura** [comment]
🤔  it's not what I see here 
screen shot 2017-03-24 at 11 32 46 am

**Comment by jbovenschen** [comment]
screen shot 2017-03-24 at 3 35 26 pm
This is what I see... Weird 😞

**Comment by diegomura** [comment]
man... I don't know. I'll then pull both branches and try to merge them locally

**Pull request merged**